client/src/components/Login/Login.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from "react";
+import "./Login.css"
+import { Form, Col, FormGroup, Button, FormControl, Well, ControlLabel } from 'react-bootstrap/lib';
+import API from '../../utils/userAPI';
+
+
+class Login extends React.Component {
+
+
+    state = {
+        username: '',
+        password: ''
+    };
+
+    handleInputChange = event => {
+        let value = event.target.value;
+        const name = event.target.name;
+
+        this.setState({
+            [name]: value
+        });
+    };
+
+    handleSubmit = event => {
+        event.preventDefault();
```

Comment:
We may want to consider removing the event.preventDefault() when we go to production and change the button to send users to the profile page after logging in.